tsconfig.json

```diff
@@ -7,10 +7,7 @@
         "forceConsistentCasingInFileNames": true,
         "skipLibCheck": true,
         "sourceMap": true,
-        "moduleResolution": "node",
-        "paths": {
-            "@ts-setup/core": ["packages/core"]
```

Comment:
This paths is not required. I am not sure why this was present. I have removed this and tested all 3 apps

Reply:
``paths`` attribute helps when you are importing in a monorepo. In my case, when I was importing stuff in vscode then a couple of times, these packages were imported as relative path files. By setting this attribute, that relative path imports problem was solved.

I don't know if I was able to explain it. I will share a link regarding it.
https://stackoverflow.com/questions/43281741/how-to-use-paths-in-tsconfig-json